datasource.js

```diff
@@ -340,7 +340,13 @@ function (angular, _, sdk, dateMath, kbn) {
     if (target.tags) {
       query.tags = angular.copy(target.tags);
       _.forOwn(query.tags, function(value, key) {
-        query.tags[key] = _.map(value, function(tag) { return self.templateSrv.replace(tag); });
+        query.tags[key] = _.flatten(_.map(value, function(tag) {
+          var replacement = self.templateSrv.replace(tag);
+          if (replacement.indexOf('{') == 0 && replacement.lastIndexOf('}') == replacement.length - 1) {
```

Comment:
this seems like a very hacky solution, KairosDB does not support glob expressions or regexes so if you want to support Multi value template variables, either KairosDB needs to support that or dont use templateSrv replace, look at the actual template variable value (it can be an array)